src/pages/SignUp/SignUp.styled.js

```diff
@@ -0,0 +1,58 @@
+import styled from 'styled-components';
+
+export const SignUpBox = styled.div`
+  display: flex;
+  justify-content: center;
+  margin: 150px 0px;
+  align-items: center;
+  height: 100%;
+`;
+
+export const SocialLoginContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  padding: 40px 16px;
+  width: 450px;
+  height: 537px;
+  margin-right: 30px;
+`;
+
+export const TextContainer = styled.div`
+  text-align: center;
+  height: 50%;
+`;
+export const IconContainer = styled.div`
+  height: 50%;
+`;
+export const Text = styled.p`
+  font-weight: ${props => (props.fontWeight ? props.fontWeight : 400)};
```

Comment:
논리곱 연산자를 활용할 수 있을 것 같습니다.
```suggestion
  font-weight: ${props => (props.fontWeight || 400)};
```

Reply:
넵! 논리합 연산자로 수정하였습니다!